bin/publish.sh

```diff
@@ -123,6 +123,8 @@ git archive --format=tar "$VERSION" \
 
 cd "$REPOSITORY"
 cp "tags/$VERSION/readme.txt" trunk/readme.txt
+mv "tags/$VERSION/assets/icon-*" assets/
```

Comment:
Do we want `mv` or `cp` here?

Reply:
I think we want mv, because we don't need them in the released version and it'll just be that much smaller…